### DISCUSSION TITLE
Issues Loading OpenStudio

### DISCUSSION THREAD

**Discussion by embar24** [urbanopt/urbanopt-cli#473] [Q&A]
Hello all! I am a new user who is trying to set up their first project. I am first running the example project to make sure everything works on my computer, but I'm getting the following error.

```
Using runner options from runner.conf file
Initializing runner with dirname: 'C:/Users/embar/Documents/UrbanOpt/Test9' and options: {:file_version=>"0.1.0", :max_datapoints=>1000000000, :num_parallel=>2, :run_simulations=>true, :verbose=>false, :gemfile_path=>"", :bundle_install_path=>""}
@bundle_without_string = ''
E, [2024-06-27T12:52:07.871016 #20384] ERROR -- : cannot load such file -- openstudio/common_measures
c:/URBANopt-cli-0.11.1/gems/ruby/2.7.0/gems/urbanopt-cli-0.11.1/lib/uo_cli.rb:25: warning: already initialized constant URBANopt::CLI::UrbanOptCLI::COMMAND_MAP
c:/URBANopt-cli-0.11.1/gems/ruby/2.7.0/gems/urbanopt-cli-0.11.1/lib/uo_cli.rb:25: warning: previous definition of COMMAND_MAP was here
Invalid command, please run uo --help for a list of available commands
```

Looking at it, there is no "openstudio/common_measures" within the Openstudio file - I used the UrbanOpt installer. Am I missing a step here?

**Comment by kflemin** [comment]
the first thing to try would be to run "bundle install" inside your project directory and ensuring that all dependencies get installed successfully.  Also, we have recently released newer versions of URBANopt which addresses many prior issues. If you are still experiencing issues related to this after upgrading, please let us know.
